app/dashboard/views.py

```diff
@@ -3936,7 +3936,7 @@ def profile(request, handle, tab=None):
         return redirect(profile.url)
 
     # setup context for visit
-    if not len(profile.tribe_members) and tab == 'tribe':
+    if not profile.tribe_members.exists() and tab == 'tribe':
```

Comment:
much more db efficient bc it just pulls a boolean back from the DB; whereas the prvious call pulled all the members back. shaves off 1s